src/Parser/Warrior/Arms/Modules/Abilities.js

```diff
@@ -1,24 +1,23 @@
 import SPELLS from 'common/SPELLS';
 
 import CoreAbilities from 'Parser/Core/Modules/Abilities';
+import ISSUE_IMPORTANCE from 'Parser/Core/ISSUE_IMPORTANCE';
 
 class Abilities extends CoreAbilities {
   spellbook() {
     return [
       {
         spell: SPELLS.MORTAL_STRIKE,
         category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
-        cooldown: 6,
-        castEfficiency: {
-          suggestion: true,
-        },
+        cooldown: haste => 6 / (1 + haste),
       },
       {
         spell: SPELLS.COLOSSUS_SMASH,
         category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
-        cooldown: 20,
-        castEfficiency: {
-          suggestion: true,
+        cooldown: (haste, combatant) => {
```

Comment:
Is CS cd reduced by haste? Your changelog entry says it is, but you don't use it here.
If it isn't, you don't need to use a function here. You can instead use the local combatant (see comment above) to get talent info. The difference between the two is that the local version snapshots your character at encounter start, so the combatant passed by the function is only needed if you need to check for a buff at a specific time during the encounter.